### PR TITLE
acpi_call-dkms: update to 1.2.1 and patch for kernel 5.13

### PR DESCRIPTION
**Pull request by dkwo**
I've tried to update as suggested #31968 , but when I install it fails to add the module
```
[*] Configuring unpacked packages
acpi_call-dkms-1.2.1_1: configuring ...
Failed to add DKMS module: 'acpi_call-1.2.1'...
acpi_call-dkms-1.2.1_1: updated successfully.
```
What should be changed?
Thanks.

### CODE REVIEW

**Comment by ericonr** [comment]
Remove the redirection to NULL in `/usr/libexec/xbps-triggers/dkms`:

```
		$DKMS add -m "$1" -v "$2" >/dev/null 2>&1
```

Then install again to see why it fails.

**Comment by dkwo** [comment]
Here's more data:
```
[*] Configuring unpacked packages
acpi_call-dkms-1.2.1_1: configuring ...
cat: VERSION: No such file or directory
dkms.conf: Error! No 'PACKAGE_VERSION' directive specified.
Error! Bad conf file.
File: /usr/src/acpi_call-1.2.1/dkms.conf
does not represent a valid dkms.conf file.
Failed to add DKMS module: 'acpi_call-1.2.1'...
acpi_call-dkms-1.2.1_1: updated successfully.
```

**Comment by dkwo** [comment]
Should I just put a `dkms.conf` file by hand in /files with 1.2.1 instead of VERSION, or is there a smarter way?

**Comment by dkwo** [comment]
This works for me, but as I said I'm open to smarter solutions :)

I guess failure is related to less, as it builds fine here..

**Comment by ericonr** [comment]
Use https://github.com/nix-community/acpi_call/pull/17 instead of vendoring in the file.

**Comment by dkwo** [comment]
Thanks. Does it look better now?

**Comment by ericonr** [comment]
It does, thanks! We can merge once things are back to normal.

File: srcpkgs/acpi_call-dkms/template [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
+checksum=aa84ac8a209e18ee6de5fa809abd3474fdca67cda3c3256f0650c4fce78d4a0f
 
 dkms_modules="acpi_call ${version}"
 depends="dkms"
 
 do_install() {
 	vmkdir usr/src/acpi_call-${version}
 	vcopy Makefile usr/src/acpi_call-${version}
-	vcopy dkms.conf usr/src/acpi_call-${version}
+	vcopy dkms.conf.in usr/src/acpi_call-${version}/dkms.conf
```
Comment (ericonr) [inline srcpkgs/acpi_call-dkms/template:20]: In `do_build` you should call `make dkms-conf`, so the `dkms.conf` file with a filled version field is generated.

Comment (dkwo) [reply srcpkgs/acpi_call-dkms/template:20]: Sure, but I need a bit more help :)
```
=> acpi_call-dkms-1.2.1_1: running do_build ...
make: *** No rule to make target 'dkms-conf'.  Stop.
=> ERROR: acpi_call-dkms-1.2.1_1: do_build: 'make dkms-conf' exited with 2
=> ERROR:   in do_build() at srcpkgs/acpi_call-dkms/template:18
```

Comment (ericonr) [reply srcpkgs/acpi_call-dkms/template:20]: Typo! `make dkms.conf` is what I meant.

Comment (dkwo) [reply srcpkgs/acpi_call-dkms/template]: Thanks.

**Pull request merged**